### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by mdellweg**

### CODE REVIEW

File: templates/github/.ci/scripts/check_release.py [hunk 1]
```diff
@@ -39,17 +25,56 @@ def main():
...
+        old_dependencies = set(old_pyproject["project"]["dependencies"])
+        new_pyproject = tomllib.load(repo.show("{to_commit}:pyproject.toml"))
+        new_dependencies = set(new_pyproject["project"]["dependencies"])
+        if old_dependencies != new_dependencies:
+            return ["dependencies"]
+        else:
+            return []
+    except Exception:
+        # Gathering more details failed.
+        return ["pyproject.toml changed somehow (PLEASE check if dependencies are affected)."]
```
Comment (pedro-psb) [inline templates/github/.ci/scripts/check_release.py:60]: Maybe adding some info about the exception could be useful to know why gathering info failed

File: plugin-template [hunk 2]
```diff
@@ -332,58 +296,27 @@ def main():
...
-
-
-def append_releasing_to_manifest(plugin_root):
-    manifest_file = Path(plugin_root) / "MANIFEST.in"
-    if manifest_file.exists():
-        manifest_text = manifest_file.read_text()
-        if "exclude releasing.md" not in manifest_text:
-            manifest_file.write_text(manifest_text + "exclude releasing.md\n")
+def migrate_dummy(plugin_root_dir):
+    pass
```
[the remark this answers is not in the document]
Comment (mdellweg) [reply plugin-template:306]: exactly. We can also remove it and refer to the git history. What do you prefer?

File: templates/bootstrap/CONTRIBUTING.md
[the remark this answers is not in the document]
Comment (mdellweg) [reply templates/bootstrap/CONTRIBUTING.md:1]: I took that file from Pulpcore.

File: templates/bootstrap/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+## Contributing
+
+We have provided detailed documentation for ways in which you can
+contribute to Pulp here:
+https://docs.pulpproject.org/contributing/
```
Comment (pedro-psb) [inline templates/bootstrap/CONTRIBUTING.md:5]:
```suggestion
https://pulpproject.org/dev/
```

Comment (pedro-psb) [reply templates/bootstrap/CONTRIBUTING.md:5]: https://github.com/pulp/pulpcore/pull/6060

**Review by pedro-psb — APPROVED** [review]

**Pull request merged**